crates/swc_cli/src/commands/mod.rs

```diff
@@ -17,9 +17,7 @@ pub enum Command {
 }
 
 #[derive(Parser)]
-#[clap(name = "SWC", version)]
-#[clap(global_setting(AppSettings::PropagateVersion))]
-#[clap(global_setting(AppSettings::UseLongFormatForHelpSubcommand))]
+#[clap(name = "SWC", version, propagate_version = true)]
```

Comment:
https://epage.github.io/blog/2022/02/clap-31-a-step-towards-40/ and happy to see clap moving towards a modular approach.